### ISSUE TITLE
`runntime error: index out of range` in test `TestInflationPF`

### ISSUE THREAD

**Issue by Charleslee522** [bosnet/sebak#817]
```
--- FAIL: TestInflationPF (15.00s)
panic: runtime error: index out of range [recovered]
	panic: runtime error: index out of range

goroutine 30 [running]:
testing.tRunner.func1(0xc000250900)
	/usr/local/go/src/testing/testing.go:792 +0x387
panic(0x875260, 0xc8ccf0)
	/usr/local/go/src/runtime/panic.go:513 +0x1b9
boscoin.io/sebak/tests/client.TestInflationPF(0xc000250900)
	/sebak-build/tests/client/congressvoting_inflationpf_test.go:99 +0x2cb8
testing.tRunner(0xc000250900, 0x910f88)
	/usr/local/go/src/testing/testing.go:827 +0xbf
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:878 +0x353
```
reference
https://circleci.com/gh/bosnet/sebak/16138?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by Charleslee522** [comment]
@kfangw Please check this test fail. I'm worried about it is related with sebak implementation.

**Comment by spikeekips** [comment]
I also met the same error in circleci, https://circleci.com/gh/bosnet/sebak/16159 .

**Comment by kfangw** [comment]
solved by #821
